erpnext/accounts/report/cost_center_wise_financial_statements/cost_center_wise_financial_statements.json

```diff
@@ -0,0 +1,30 @@
+{
+ "add_total_row": 0,
+ "creation": "2020-10-20 00:25:11.717192",
+ "disable_prepared_report": 0,
+ "disabled": 0,
+ "docstatus": 0,
+ "doctype": "Report",
+ "idx": 0,
+ "is_standard": "Yes",
+ "modified": "2020-10-20 00:25:11.717192",
+ "modified_by": "Administrator",
+ "module": "Accounts",
+ "name": "Cost Center Wise Financial Statements",
+ "owner": "Administrator",
+ "prepared_report": 0,
+ "ref_doctype": "GL Entry",
+ "report_name": "Cost Center Wise Financial Statements",
+ "report_type": "Script Report",
+ "roles": [
+  {
+   "role": "Accounts User"
+  },
+  {
+   "role": "Accounts Manager"
+  },
+  {
+   "role": "Auditor"
+  }
+ ]
```

Comment:
Should any other users be given access to the document? Administrator for example (if not done by default)

Reply:
@rvpasari  administrator has all role by default.  this is account related report should be access by accounts role